Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -59,6 +62,13 @@ public final class CommentCellViewModel:
 
     self.authorName = comment.map(\.author.name)
 
+    let status = comment.map(\.status)
+
+    self.commentStatus = Signal.merge(
```

Comment:
Does `self.commentStatus = comment.map(\.status).signal.takeWhen(self.bindStylesProperty.signal)` achieve the same desired effect?

Reply:
I think if indented appropriately the readability is good too but I didn't test out whether the syntax actually results in the functionality you want here.

Reply:
We want to always have an initial status before the `self.bindStylesProperty` is called, should there be a need for reset e.g screen orientation change.